docs/trace/getting-started-jaeger/README.md

```diff
@@ -147,7 +151,11 @@ using var tracerProvider = Sdk.CreateTracerProviderBuilder()
     ...
     // Remove Console Exporter from the final application
     // .AddConsoleExporter()
-    .AddJaegerExporter()
+    .AddOtlpExporter(opt =>
+    {
+        opt.Endpoint = new Uri("http://localhost:4317");
+        opt.Protocol = OtlpExportProtocol.Grpc;
+    })
```

Comment:
I have checked PR once more time. These values looks like defaults.
I think it could be simplified, but it is not blocking change.

```suggestion
    .AddOtlpExporter()
```

Reply:
Thanks Kielek! I've removed these redundant option settings and updated the `README`.